generate_package_xml_recursive_dependencies.py

```diff
@@ -97,7 +97,7 @@ def main():
     meta_package = PackageTemplate._create_package_template(
         package_name=f'meta-{args.variant}', licenses=["GPLv3"]
     )
-    if args.rosdistro in ["foxy", "humble"]:
+    if args.rosdistro in ["foxy", "humble", "jazzy"]:
```

Comment:
Could you change this for a `!= "noetic"` so that we don't have to keep adding new ROS 2 distro.